lib/Test/Selenium/Remote/Driver.pm

```diff
@@ -236,65 +240,170 @@ more documentation, see the related test methods in L<Selenium::Remote::Driver>
     click_ok
     double_click_ok
 
-=head2 $twd->type_element_ok($search_target, $keys, [, $desc ]);
+=head2 $twd->type_element_ok($search_target [,$locator], $keys, [, $desc ]);
 
-   $twd->type_element_ok( $search_target, $keys [, $desc ] );
+   $twd->type_element_ok( $search_target [,$locator], $keys [, $desc ] );
 
 Use L<Selenium::Remote::Driver/find_element> to resolve the C<$search_target>
-to a web element, and then type C<$keys> into it, providing an optional test
+to a web element and an optional locator, and then type C<$keys> into it, providing an optional test
 label.
```

Comment:
Want to move this pod for type_element_ok to the other side of _find_element_with_action, so it's near the implementation of type_element_ok ?